lib/lotus/cli.rb

```diff
@@ -1,110 +1,134 @@
+# TODO: Clarify/Questions
+# Create a migration along with the entity?
+# Reneame generate model => generate entity?
+# Naming slice/apps. I'd keep the slice generator, drop the app generator (slice can do everything app can)
```

Comment:
ok, so'll do the opposite:-) keep "app", remove "slice". I know, from the directory structure it's "apps" anyways. But it sometimes confusing whether it's the "app" (the whole lotus application, generated with "lotus new") or one of the "apps", that is a sub-app. but i agree, slice is not well known terminology, i actually only found it in the generator.